src/crypto/mod.rs

```diff
@@ -5,8 +5,6 @@
 pub mod error;
 mod into_fr;
 pub mod poly;
-#[cfg(feature = "serialization-protobuf")]
```

Comment:
Do you think `serialization-protobuf` might stay in the `threshold_crypto` crate?

Reply:
I'd vote for removing it there, too, but if we make sure the implementation is complete and functional, I'm happy with keeping it, too.

(In this PR, I guess, I can remove it, since the whole module will be moved out either way?)

Reply:
Please keep the change here. I'll take care of in in the other crate. We have to remove `protobuf_impl.rs` as well.

Reply:
Right, good catch! I totally missed that!